src/Operation/Aggregate.php

```diff
@@ -299,7 +299,11 @@ private function createCommand(Server $server)
 
         $cmd['allowDiskUse'] = $this->options['allowDiskUse'];
 
-        if (isset($this->options['bypassDocumentValidation']) && \MongoDB\server_supports_feature($server, self::$wireVersionForDocumentLevelValidation)) {
+        if (
+            isset($this->options['bypassDocumentValidation']) &&
+            $this->options['bypassDocumentValidation'] &&
+            \MongoDB\server_supports_feature($server, self::$wireVersionForDocumentLevelValidation)
+        ) {
```

Comment:
Optional: refactor `isset($x) && $x` as `!empty($x)`. There's some prior art for this in the library (and in `Aggregate.php` in fact).